src/components/screens/registerDelegate/selectName/SelectName.js

```diff
@@ -130,7 +131,12 @@ class SelectName extends React.Component {
             }
           </p>
 
-          <label className={styles.nicknameLabel}>{t('Your nickname')}</label>
+          <label className={styles.nicknameLabel}>
+            {t('Your nickname')}
+            <Tooltip classNam>
```

Comment:
Is `classNam` a typo?

Reply:
the autocomplete feature messing with me. Yes it is a typo thanks for the catchup Ali